torch/distributed/distributed_c10d.py

```diff
@@ -555,7 +555,7 @@ class group(metaclass=_WorldMeta):
     pass
 
 class GroupMember(metaclass=_WorldMeta):
-    NON_GROUP_MEMBER = object()
+    NON_GROUP_MEMBER = -100
```

Comment:
comparison w/ object NYI - -100 is spiritually okay (any identity)

Reply:
hmm.. yes, looks ok 👍

Reply:
This one also seems simple to fix directly in Dynamo